book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    @rentals = Rental.new(date, person, self)
+  end
+
```

Comment:
- [ ] If you check you can see that the rental class already add the rental to the `rentals` array so this would be repetitive.

- [ ] Also remember to arrange your parameters based on how you defined it in the Rental constructor: `self` should come before `person`

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```